--- /tmp/olddocker.py	2015-08-26 14:27:35.102805655 -0400
+++ /usr/lib/python2.7/site-packages/ansible/modules/core/cloud/docker/docker.py	2015-08-26 12:20:22.594770205 -0400
@@ -352,6 +352,7 @@
 '''
 
 HAS_DOCKER_PY = True
+DEFAULT_DOCKER_API_VERSION = None
 
 import sys
 import json
@@ -454,6 +455,7 @@
                     if not char.isdigit():
                         nondigit = part[idx:]
                         digit = part[:idx]
+                        break
                 if digit:
                     version.append(int(digit))
                 if nondigit:
